gitignore

```diff
@@ -1,21 +1,49 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-if [ -z "${1:-}" ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
-    echo -e "Generate .gitignore file for the language you prefer.\nUses https://gitignore.io\n"
+if [ -z "${1:-}" ] || [[ ${@} =~ "," ]] || printf '%s\n' ${@} | grep -q -P '^--?h(elp)?$'
+then
+    echo "Generate .gitignore file for the language you prefer."
+    echo -e "Uses https://gitignore.io\n"
     echo "Usage:"
-    echo -e "  gitignore <LANGUAGE>\n"
-    echo "For example:"
-    echo "  gitignore python"
+    echo -e "  gitignore <environments> \n"
+    echo "Example:"
+    echo -e " for python:"
+    echo -e "  gitignore python \n"
+    echo -e " for macos, pycharm and python:"
+    echo -e "  gitignore macos python pycharm \n"
+    echo -e " for pycharm:"
+    echo -e "  gitignore pycharm \n"
     exit 1
 fi
 
-response="$(curl -sL https://www.gitignore.io/api/${1})"
+# implode all arguments by using ',' as delimiter and remove trailing ','
```

Comment:
what does `implode all arguments` mean?

Reply:
glue all the user given inputs with the delimiter as ','